src/Core/Domain/Product/ValueObject/Isbn.php

```diff
@@ -38,7 +38,7 @@ class Isbn
     /**
      * Valid ISBN regex pattern
      */
-    public const VALID_PATTERN = '/^[0-9-]{0,32}$/';
+    public const VALID_PATTERN = '/^[0-9][0-9-]{1,30}[0-9X]$/i';
```

Comment:
This regex is not valid regarding the https://en.wikipedia.org/wiki/International_Standard_Book_Number
```suggestion
    public const VALID_PATTERN = '/^(?=(?:\D*\d){10}(?:(?:\D*\d){3})?X?$)[\d-X]+$/i';
```
Are you looking for something like that?